### DISCUSSION TITLE
Substitution of the device name

### DISCUSSION THREAD

**Discussion by frizib** [PrismarineJS/bedrock-protocol#227] [Q&A]
![image](https://user-images.githubusercontent.com/28644006/172431078-6a06059e-5526-42d5-bd49-3da9d334f30c.png)
Good days.

I saw here the topic of how to change the operating system.

Is it possible to change the name of the device somehow?

For example, "Xiaomi Redmi Go"

**Comment by extremeheat** [comment]
Going to assume this is a question about using proxy. You need to find correct packet (I believe the login one) and modify the fields. Moving to discussion as not an issue with library
